setup/kubernetes/local-preview.md

```diff
@@ -4,8 +4,8 @@ description: Set up a Coder deployment locally for testing.
 ---
 
 Coder is typically deployed onto Kubernetes cluster, but if you would like to
-set up a lightweight preview deployment, you can do so locally using using
-[Docker][docker-url] and [kind][kind-url].
+set up a lightweight preview deployment, you can do so locally using using [URL]
+and [URL].
```

Comment:
I'm a bit confused how these `[URL]` links work

Reply:
they don't: my linter messed that up. i'll fix